### DISCUSSION TITLE
Source code structure to integrate the easy-cucumber with a repository that already using original cucumber in it

### DISCUSSION THREAD

**Discussion by shoevancar** [CXwudi/comp5903-easy-cucumber#21] [Q&A] [answered]
Would you please help me understand the following points?

(1) Do I need to re-write the same feature files, and step definitions for easy-cucumber? or we can just build and run the existing features (correction grid of 56 acceptance tests) written for the original cucumber?

(2) Do I need to create a separate repository to segregate the changes made for easy cucumber? or I can keep writing committing code in one single repository.

**Comment by CXwudi** [comment, marked as answer]
1) as long as you only used [supported features](https://github.com/CXwudi/comp5903-easy-cucumber#supported-features), then you can reuse the same feature file.

2) one repo is fine, I believe TDD is not required in A2, wasn't it?

**Reply by shoevancar** [reply]
Thank you so much for your reply.

Yes, TDD is not required for A2.
